afs/afs-mapdb-storage/src/main/java/com/powsybl/afs/mapdb/storage/MapDbAppStorage.java

```diff
@@ -261,6 +274,8 @@ public NodeInfo createRootNodeIfNotExists(String name, String nodePseudoClass) {
             rootNodeInfo = createNode(null, name, nodePseudoClass, "", 0, new NodeGenericMetadata());
             rootNodeVar.set(rootNodeInfo);
         }
+        this.enable(rootNodeInfo.getId());
```

Comment:
Can be done in the top if { }, only after node creation?

Reply:
we do that only if the rootNode is created .